runtime/onert/core/src/ir/OpSequences.cc

```diff
@@ -30,12 +30,46 @@ OpSequenceIndex OpSequences::emplace(const OperationIndex &index, Layout layout)
 {
   std::unique_ptr<OpSequence> op_seq = std::make_unique<OpSequence>(layout);
   op_seq->appendOperation(index);
-  return push(std::move(op_seq));
+  const OpSequenceIndex &seq_index = push(std::move(op_seq));
+  cacheSequenceIndex(seq_index, index);
+  return seq_index;
 }
 
 OpSequenceIndex OpSequences::emplace(std::unique_ptr<OpSequence> &&op_seq)
 {
-  return push(std::move(op_seq));
+  auto &operations = op_seq->operations();
+  const OpSequenceIndex &seq_index = push(std::move(op_seq));
+  for (const auto &op_idx : operations)
+  {
+    cacheSequenceIndex(seq_index, op_idx);
+  }
+  return seq_index;
+}
+
+void OpSequences::cacheSequenceIndex(const OpSequenceIndex &seq_index,
+                                     const OperationIndex &op_index) const
+{
+  _seq_indexes.insert(
+      std::pair<const OperationIndex &, const OpSequenceIndex &>(op_index, seq_index));
```

Comment:
@periannath You may use `_seq_indexes.emplace(op_index, seq_index)`.

Reply:
@glistening Thanks! I'll update it.